File: easy_roman.py
def roman(s):
    
    RomanDic = {"I":1,"V":5,"X":10,"L":50,"C":100,"D":500,"M":1000}
    result = 0
    for i in range(len(s)-1):
        if RomanDic[s[i]]<RomanDic[s[i+1]]:
            result -= RomanDic[s[i]]
        else:
            result += RomanDic[s[i]]
    result += RomanDic[s[-1]]
    return result

s=input("Enter a Roman Numeral: ")
r=roman(s)
print(r)
#XXIV
#10
#20
#20-1=19
#19+5
#24

File: easy_roman.java
import java.util.*;
class Solution{
	public int roman(String s){
		HashMap<Character, Integer> map = new HashMap<>(); 
		map.put('I',1);
		map.put('V',5);
		map.put('X',10);
		map.put('L',50);
		map.put('C',100);
		map.put('D',500);
		map.put('M',1000);
		int result=0;
		for (int i=0; i<s.length()-1; i++){
			if (map.get(s.charAt(i))<map.get(s.charAt(i+1))){
				result-= map.get(s.charAt(i));
			}
			else{
				result+=map.get(s.charAt(i));
			}
		}
		result+=map.get(s.charAt(s.length()-1));
		return result;
	}
}

class easy_roman{
	public static void main(String [] args){
		String s= "XXIV"; 
		Solution x= new Solution();
		System.out.println(x.roman(s));
	}
}